src/seedu/addressbook/commands/SortCommand.java

```diff
@@ -0,0 +1,26 @@
+package seedu.addressbook.commands;
+
+import seedu.addressbook.data.person.ReadOnlyPerson;
+
+import java.util.List;
+
+
+/**
+ * Lists all persons in the address book to the user.
```

Comment:
Incorrect header comment. Take care to update the comments (this seems like a copy-paste-forgot problem)